ios/RNSScreenStackAnimator.mm

```diff
@@ -105,28 +105,40 @@ - (void)animateSimplePushWithTransitionContext:(id<UIViewControllerContextTransi
     leftTransform = CGAffineTransformMakeTranslation(belowViewWidth, 0);
   }
 
+  auto shadowView = [[UIView alloc] initWithFrame:fromViewController.view.frame];
+  shadowView.backgroundColor = [UIColor blackColor];
+
   if (_operation == UINavigationControllerOperationPush) {
     toViewController.view.transform = rightTransform;
     [[transitionContext containerView] addSubview:toViewController.view];
+    [[transitionContext containerView] insertSubview:shadowView belowSubview:toViewController.view];
+    shadowView.alpha = 0.0;
+
     [UIView animateWithDuration:[self transitionDuration:transitionContext]
         animations:^{
           fromViewController.view.transform = leftTransform;
           toViewController.view.transform = CGAffineTransformIdentity;
+          shadowView.alpha = 0.1;
```

Comment:
My only suggestion here would be to extract this target alpha value to a constant, see the top of the `RNSScreenStackAnimator.mm` file (maybe use `constexpr` instead of static const) & if there is any reasoning behind this value (e.g. "Looked best in experimentation") - leave a comment behind.

Reply:
Sure thing, I'll extract it and I'll add a comment a comment with explanation